test-gui.sh

```diff
@@ -36,7 +36,7 @@ then
 fi
 
 yarn install
-yarn run webdriver-update
+yarn run webdriver-update-protractor
```

Comment:
```suggestion
export CHROME_VERSION=$("$CHROME_BINARY_PATH" --version)
yarn run webdriver-update --versions.chrome="$CHROME_VERSION"
```

Reply:
just trying a test, only new additions are 'cypress' and 'eslint-plugin-cypress`, searched thru `node_modules` don't see anyone using webdriver other than protractor.

Reply:
@dtaylor113 I'll probably open another PR to address our Chrome version handling. I think trying to download on CI runs is no longer necessary and is actually causing more problems than it fixes.

Reply:
@spadgett , I'm just debugging trying to figure out why this is happening in this PR and not others.

Reply:
#5448 should address the problem regardless

Reply:
#5448 did fix the issue! :-)